contents/handbook/people/hiring-process/index.md

```diff
@@ -22,6 +22,14 @@ This has resulted in the highest number of qualified and motivated candidates re
 - Giving candidates genuinely useful and direct feedback if they weren't successful with us
 - Running a smooth and incredibly slick recruitment process, from application to offer
 
+## Countries we employ people in
+
+We are all-remote, but we have a few limitations on the countries we are able to employ people in:
+
+- We hire within -8 to +2 GMT timezones - beyond these, it proves too difficult for us to collaborate effectively
```

Comment:
Would we be okay with Estonia or Greece, which are GMT+3 in the summer?

Reply:
Yes I think so - do you reckon this needs to be called out? (We don't screen out people from those countries on the ops team at least)

Reply:
Could prevent some of these good candidates from applying. Maybe just:
```suggestion
- We hire within -8 to +3 GMT timezones - beyond these, it proves too difficult for us to collaborate effectively
```